src/ontology/uberon-edit.obo

```diff
@@ -222488,6 +222488,17 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500108
+name: cheek tooth
+def: "Any of the post-canine teeth, which are the multi-cusped molars or premolars used for shearing and grinding food in mammals." [Wikipedia:Cheek_teeth]
+synonym: "cheek teeth" RELATED [] {synonymtypedef="PLURAL"}
```

Comment:
I think the refs:label is 'plural term'.
See UBERON:0004865 in Protégé as an example.